modules/lm-coursier/src/main/scala/lmcoursier/CoursierConfiguration.scala

```diff
@@ -111,5 +172,53 @@ object CoursierConfiguration {
       credentials,
       Option(logger),
       Option(cache),
+      Option(ivyHome),
+      Option(followHttpToHttpsRedirections),
+      Option(strict),
+      extraProjects,
+      forceVersions,
+      reconciliation,
+      classpathOrder,
+      verbosityLevel,
+      Some(ttl),
+      checksums,
+      cachePolicies,
     )
+
+  private[lmcoursier] def cachePolicyToString(policy: CachePolicy): String = {
```

Comment:
There's already [a method](https://github.com/coursier/coursier/blob/904670c8ab47c7fbc4c9c28aeeb4a63a71ce80d9/modules/cache/jvm/src/main/scala/coursier/parse/CachePolicyParser.scala#L9) going from `String` to `CachePolicy` in coursier. I guess we can temporarily add one going from `CachePolicy` to `String` here using the same names, until it's added in coursier itself.

Reply:
But this method shouldn't be necessary if we use an ADT for `CachePolicy`.

Reply:
true. Will pursue that indeed. We'll still need some form of that, as my goal is to pick the defaults from CoursierDefaults (where Coursier sets the default policy), I'll need to have a conversion from the `coursier.CachePolicy` to the `lmcoursier.definition.CachePolicy`. But it should be enough indeed.